helm_deploy/values-preprod.yaml

```diff
@@ -34,3 +34,6 @@ generic-service:
     SHOW_PRISON_TRANSFER_SUMMARY: 'false'
     SHOW_BREADCRUMB: 'true'
     SHOW_RECENT_ARRIVALS: 'true'
+
+generic-prometheus-alerts:
```

Comment:
This label align with pre-prod env promoethus alerts and are introduced new in the WPIP-UI